types/react-sound/index.d.ts

```diff
@@ -6,29 +6,32 @@
 
 import * as React from "react";
 
-declare namespace ReactSound {
-    type PlayStatus = 'PLAYING' | 'STOPPED' | 'PAUSED';
+export enum PlayStatus {
+    Playing = 'PLAYING',
+    Stopped = 'STOPPED',
+    Paused = 'PAUSED'
+}
 
-    interface ReactSoundProps {
-        url: string;
-        playStatus: PlayStatus;
-        playFromPosition?: number;
-        position?: number;
-        volume?: number;
-        playbackRate?: number;
-        autoLoad?: boolean;
-        loop?: boolean;
-        onError?: () => void;
-        onLoading?: () => void;
-        onLoad?: () => void;
-        onPlaying?: () => void;
-        onPause?: () => void;
-        onResume?: () => void;
-        onStop?: () => void;
-        onFinishedPlaying?: () => void;
-        onBufferChange?: () => void;
-    }
+export interface ReactSoundProps {
+    url: string;
+    playStatus: PlayStatus;
+    playFromPosition?: number;
+    position?: number;
+    volume?: number;
+    playbackRate?: number;
+    autoLoad?: boolean;
+    loop?: boolean;
+    onError?: () => void;
+    onLoading?: () => void;
+    onLoad?: () => void;
+    onPlaying?: () => void;
+    onPause?: () => void;
+    onResume?: () => void;
+    onStop?: () => void;
+    onFinishedPlaying?: () => void;
+    onBufferChange?: () => void;
 }
 
-declare const ReactSound: React.ComponentClass<ReactSound.ReactSoundProps>;
-export = ReactSound;
+declare const ReactSound: React.ComponentClass<ReactSoundProps>;
+
+export default ReactSound;
```

Comment:
Correct, `require("react-sound")` is an object with a `default` property and doesn't directly export the function.